services/semaphore.js

```diff
@@ -1,3 +1,4 @@
+const parseGitUrl = require('git-url-parse');
```

Comment:
I prefer to not add any dependency. We used to depend on `git-url-parse` in semantic-release and we removed it after having many issues.

Reply:
Given the many potential forms of a git URL, I am not willing to parse it myself (won't do a better job than `git-url-parse`) . I'll remove the slug from v2. (Referring to your comment below)

> If there is no simple way to get the `slug` (like `SEMAPHORE_REPO_SLUG`) then don't add it. It will be `undefined` and will be removed from the doc.